app/models/daos/user.dao.js

```diff
@@ -0,0 +1,8 @@
+// development dependencies
+var DA0 = require('./dao-factory');
+var db = require('../models/config/database');
+var userParams = require('../schema/user.json');
+
+db.connect();
+
+var UserDAO = DAO("UserDAO", client, userParams);
```

Comment:
Mixed double and single quotes.<br>'DAO' is not defined.<br>'client' is not defined.<br>'UserDAO' is defined but never used.